fixtures/ssr/src/components/App.js

```diff
@@ -10,23 +10,29 @@ function LoadingIndicator() {
   return <div className={theme + '-loading'}>Loading...</div>;
 }
 
-export default function App({assets}) {
+function Content() {
```

Comment:
I extracted this so that you can add a `throw` in this to test the error behavior. You can't throw in App because there's no Suspense around it.